actions/deploy-tag-update/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -eu
+
+cd datarepo-helm-definitions
+
+# determine whether repo is api or ui
+case "${PWD}" in
+    *-ui) GIT_WHICH_REPO=ui ;;
+    *) GIT_WHICH_REPO=api ;;
+esac
+
+# get latest semantic version
+REPO_TAG=$(git describe --tags --abbrev=0)
+
+# get current helm image tag version
+HELM_DEV=$(find . -name devDeployment.yaml)
+HELM_PROP="datarepo-${GIT_WHICH_REPO}.image.tag"
+HELM_TAG=$(yq r "${HELM_DEV}" "${HELM_PROP}")
+
+# replace the image tag if it doesn't match the semantic version
+if [ "${REPO_TAG}" != "${HELM_TAG}" ]; then
+    yq w -i "${HELM_DEV}" "${HELM_PROP}" "${REPO_TAG}"
+    # configure git settings
+    git config --global user.email "broadbot@broadinstitute.org"
+    git config --global user.name "broadbot"
+    git config pull.rebase false
+    # commit changes to helm definitions
+    git pull origin master
+    git add *.yaml
```

Comment:
could you use the env var here?
```suggestion
    git add "${HELM_DEV}"
```

Reply:
Yes, you're right, I copied this over from the other action. I'll fix this as well.